artifacts/centosstream/centos-stream_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("CentosStream", func() {
 	DescribeTable("Inspect should be able to parse checksum files",
 		func(release, mockFile string, details *api.ArtifactDetails,
 			exampleUserData *docs.UserData, envVariables map[string]string, metadata *api.Metadata) {
-			c := New(release, exampleUserData, envVariables)
+			c := New(release, "x86_64", exampleUserData, envVariables)
```

Comment:
please change this to constant

Reply:
We don't have constants of other values, why this one should be?